Makefile

```diff
@@ -24,3 +24,9 @@ test:
 license:
 	./scripts/add_license.sh
 
+build:
+	cd cmd/retrieval-client && go build -o ../build/retrieval-client .
```

Comment:
this places the binary in `cmd/build`, is that intended?

Reply:
FIxed :)